static/app/components/feedback/useMutateActivity.tsx

```diff
@@ -79,23 +79,23 @@ export default function useMutateActivity({
 
   const handleUpdate = useCallback<UpdateCommentCallback>(
     (note, noteId, activity, options) => {
-      mutation.mutate([{note, noteId, activity}, 'PUT'], options);
+      mutate([{note, noteId, activity}, 'PUT'], options);
     },
-    [mutation]
+    [mutate]
```

Comment:
top-level objects returned by `useQuery` and `useMutation` are _not_ referentially stable, but functions and values on them are. The linter knows this and now recommends destructuring those.

Essentially, the memoization with `useCallback` didn’t do anything before, so I’m wondering why we are having it in the first place?